### PR TITLE
Update code sample on about page with ES6 features

### PR DESCRIPTION
**Pull request by Chris911**
Small update on the about page to use ES6 features in code sample supported on on Node latest.

Looks like:
screen shot 2015-11-26 at 12 55 05 am

### CODE REVIEW

**Comment by fhemberger** [comment]
Nice, LGTM

**Comment by rvagg** [comment]
maybe it's time to do a proper job and introduce template strings at the same time?

``` js
const http = require('http');

http.createServer((req, res) => {
  res.writeHead(200, { 'Content-Type': 'text/plain' });
  res.end('Hello World\n');
}).listen(1337, '127.0.0.1', function () {
  console.log(`Server running at http://${this.address().address}:${this.address().port}/`);
});
```

or is that too complicated?

**Comment by fhemberger** [comment]
@rvagg Good point! Maybe add:

``` js
const connection = this.address();
console.log(`Server running at http://${connection.address}:${connection.port}/`);
```

**Comment by ghost** [comment]
LGTM, regarding the complexity, maybe link to the ES6 page right below the code block? something like `This code example uses EcmaScript 2015 (ES6) features. Read more _here_`

**Comment by Chris911** [comment]
@rvagg, @fhemberger, I can make the change the add the template string but personally I think it adds complexity by using `this.address().address`. Since this is a small introduction snippet maybe something like: 

``` js
const http = require('http');

const HOSTNAME = '127.0.0.1';
const PORT = 1337;

http.createServer((req, res) => {
  res.writeHead(200, { 'Content-Type': 'text/plain' });
  res.end('Hello World\n');
}).listen(PORT, HOSTNAME, function () {
  console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
});
```

This also matches the terminology used in the [`server.listen` documentation](https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback).

**Comment by fhemberger** [comment]
@Chris911 Good idea, let's settle for this.

**Comment by Chris911** [comment]
@fhemberger updated.

**Comment by stevemao** [comment]
:+1:

File: locale/en/about/index.md [hunk 1]
```diff
@@ -11,14 +11,17 @@ connections can be handled concurrently. Upon each connection the callback is
...
-  res.end('Hello World\n');
-}).listen(1337, '127.0.0.1');
+const HOSTNAME = '127.0.0.1';
+const PORT = 1337;
 
-console.log('Server running at http://127.0.0.1:1337/');
+http.createServer((req, res) => {
+  res.writeHead(200, { 'Content-Type': 'text/plain' });
+  res.end('Hello World\n');
+}).listen(PORT, HOSTNAME, function () {
```
Comment (phillipj) [inline locale/en/about/index.md]: Last nitpick, use arrow function on this callback aswell?

**Comment by rvagg** [comment]
I'd vote for lower-case variable names, there's inconsistency in `http` vs `PORT` and `HOSTNAME` and upper-casing constants is not necessarily idiomatic Node or even JavaScript even though it's common in other languages.

File: locale/en/about/index.md (hunk 1 above)
Comment (Chris911) [inline locale/en/about/index.md]: Good catch.

**Comment by Chris911** [comment]
Fine with me @rvagg.

**Comment by Chris911** [comment]
Updated.

**Comment by phillipj** [comment]
Nice :+1:

**Pull request merged**